node-commons/src/assembly/resources/conf/iotdb-common.properties

```diff
@@ -997,7 +997,6 @@
 # config_node_ratis_snapshot_trigger_threshold=400000
 # schema_region_ratis_snapshot_trigger_threshold=400000
 # data_region_ratis_snapshot_trigger_threshold=400000
-# config_node_simple_consensus_snapshot_trigger_threshold=400000
```

Comment:
why delete trigger threshold?

Reply:
`config_node_simple_consensus_trigger_threshold` seems to be mistakenly placed under ratis configuration section. It is not used anywhere in code, so I deleted it.